src/renderer/themes.css

```
@@ -50,13 +50,6 @@ body:not(.hotPink, .pastelPink) {
 .system[data-system-theme*='dark'],
 .dark,
 .black,
```

Looks like an unintentional removal:

```suggestion
.black {
  --logo-icon: url('../../_icons/iconColorSmall.svg');
  --logo-text: url('../../_icons/textColorSmall.svg');
  --logo-primary-color: #f33;
  --logo-secondary-color: #29abe1;
  --logo-tertiary-color: #000;
}
```

---

uhh no? This was within the gray block?

---

Not just a gray block:
https://github.com/FreeTubeApp/FreeTube/blob/47d4103911252b20b75b6dce52026150981f10b4/src/renderer/themes.css#L47-L59

---

You got rid of the block, not just the `, .gray`

---

ahhhhh sorry i dont know how i overlooked that